Command/DumpCommand.php

```diff
@@ -86,7 +87,11 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     {
         parent::initialize($input, $output);
 
-        $this->targetPath = $input->getArgument('target') ?: sprintf('%s/../web/js', $this->kernelRootDir);
+        $this->targetPath = $input->getArgument('target') ?: sprintf(
+            '%s/../%s/js',
+            $this->kernelRootDir,
+            Kernel::VERSION_ID >= 40000 ? 'public' : 'web'
```

Comment:
Not all Symfony 4 projects would use `public`, E.G if you upgrade a SF3 project to SF4 but decide to keep the `web` directory instead of changing it to `public

Reply:
better would be to detect the fact that this is a Flex project, using `class_exists(Recipe::class)` (with the right use statement)

Reply:
On looking at it, I am wondering if it isn't better still to do a `file_exists()`, as even if it isn't a Flex project it could still be in either. Thoughts?

Reply:
I would not do a file_exists, as this might create issue (for instance, if you do it on the `js` subfolder, it might not exist yet).
This is about the default behavior anyway. So anyone using a custom structure can already pass the `target` argument explicitly.

Reply:
I was thinking checking the web vs public, but fair point regardless. :+1: